src/nn/ops/unary.ts

```diff
@@ -18,12 +19,19 @@ export abstract class Unary extends SingleOutputOperation {
       this.x_ = undefined;
     }
     this.needCheckOutputShape_ = true;
+    this.createOutput();
   }
 
   inputs(): MLOperand[] {
     return [this.x_];
   }
 
+  createOutput(): void {
+    if (this.x_) {
```

Comment:
Should `this.x_` always be valid?

Reply:
No, here is for those activation functions such as `relu()` without input.